SPECK/src/main/java/SPEck.java

```diff
@@ -25,28 +25,43 @@
  */
 
 public class SPEck implements Serializable {
+
+    /**
+     * Initializer
+     * @param   scc    <JavaSparkContext> spark context for parallel execution
+     * @param   fileIn  <String> storing the file name of the input dataset to mine the SFSP
+     * @param   fileOut <String> storing the file name of the output (SFSP)
```

Comment:
Similar comment as above about "filename", and let's rename the variable to `outSfspPath`.